### PR TITLE
[Windows] Adds note about the MSVC DLLs to the instructions

### PR DESCRIPTION
**Pull request by mrry**
The standard Python.org version of Python 3.5 does not include the MSVC redistributable, on which TensorFlow depends. Add a note to the docs with the error message and a remedy.

### CODE REVIEW

File: tensorflow/g3doc/get_started/os_setup.md [hunk 1]
```diff
@@ -136,12 +136,18 @@ You can now [test your installation](#test-the-tensorflow-installation).
...
 
-* [Python 3.5 from python.org](https://www.python.org/downloads/release/python-352/)
 * [Python 3.5 from Anaconda](https://www.continuum.io/downloads#windows)
 
+* [Python 3.5 from python.org](https://www.python.org/downloads/release/python-352/). (NOTE:
+  you may need to install the [Visual C++ 2015
+  redistributable](https://www.microsoft.com/en-us/download/details.aspx?id=53587)
+  (x64 version) when using this version of Python. If you see an error message that says
+  "ImportError: DLL load failed: The specified module could not be found", then you
+  must install the redistributable.)
```
Comment (wolffg) [inline tensorflow/g3doc/get_started/os_setup.md]: Maybe:  [separate line:]

> NOTE:  TensorFlow requires `msvcp140.dll`. 
> If, when you `import tensorflow` you see an error about "No module named "_pywrap_tensorflow"' and/or
> "DLL load failed", check for  msvcd140.dll in `C:\Windows\System32\` and elsewhere.  
> If it is missing, you may need 
> to install the [Visual C++ 2015 redistributable](https://www.microsoft.com/en-us/download/details.aspx?id=53587) (x64 version).

**Review by wolffg — COMMENTED** [review]
PTAL for alternate wording.

**Comment by mrry** [comment]
That's better thanks. I applied a (slightly modified) version of it... PTAL.

**Comment by mrry** [comment]
Hopefully this wording will serve as a fix for #5949.

**Review by wolffg — APPROVED** [review]
LGTM

**Comment by beyhangl** [comment]
@mrry TensorFlow successfully installed.
The installed version of TensorFlow does not include GPU support.
I have installed CUDA already so why ?

**Comment by mrry** [comment]
@beyhangl I'm not sure how you installed TensorFlow. However, if you used the `pip install` command, you need to call `pip install tensorflow-gpu` to get the GPU version; `pip install tensorflow` will install the CPU-only version.

**Comment by beyhangl** [comment]
@mrry  yeah already installed with 

> pip install tensorflow-gpu 

I don't know why.When writing  `

> import tensorflow as tf

`It doesn't say 'i have already seen your gpu bro' , i mean gpu

**Pull request merged**